irods-vfs-impl/src/main/java/org/irods/nfsrods/vfs/ServerMain.java

```diff
@@ -2,6 +2,8 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.Map;
```

Comment:
Remove?

Reply:
Yep, I originally had it in there b/c I was playing around with environment variables, but forgot to delete the imports.